### PR TITLE
New Spanish language added via upload

### PR DESCRIPTION
**Pull request by antoromani**
New Spanish language added, please review

### CODE REVIEW

File: locales/es/front/credits.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+  },
+  "formOperation": {
+    "label": {
+      "DealWithAccounts": "Afecta las cuentas",
+      "LinkToPlan": "Enlace al plan",
+      "LinkToPlanItem": "{date}: {paid} {ofTotal} ({percent}%)",
+      "NoLinkToPlan": "fuera del plan",
+      "SelectTransaction": "Seleccione transacción...",
+      "TransactionLinked": "Transacción vinculada por {amount} en {date}{ifAccountValid, seleccione, 0{ with another account} otra{}}",
```
Comment (Vovan-VE) [inline locales/es/front/credits.json]: Please, check ICU `{select}` syntax: https://formatjs.io/docs/core-concepts/icu-syntax/#select-format
There are several `{select}`s is all the files.

File: locales/es/front/receipts.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      "PositionPriceVary": "la posición dell precio varía",
+      "priceDifference": "Diferencia",
+      "Shopping": "Compras",
+      "unknownShop": "tienda desconocida"
+    },
+    "message": {
+      "ConfirmDeleteReceipt": "¿Está seguro de eliminar este recibo?",
+      "LinkedTransactionsWillLeft": "Las transacciones vinculadas ({count}) no se eliminarán.",
+      "NoPositions": "Sin posiciones",
+      "SelectPositionsToTrack": "Seleccione {maxCount, plural, =1{a position} uno{up to # position} otro{up to # positions}} para ver precios."
```
Comment (Vovan-VE) [inline locales/es/front/receipts.json]: Check also `{plural}` syntax: https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format
There are several `{plural}`s is all the files.

**Review by Vovan-VE — CHANGES_REQUESTED** [review]
It's my fail to not mention ICU syntax in README. Sorry for that.

In general, all the "keys" in `select` and `plural` must be untouched, and all of then must include `other` key, even if empty `other{}`, or formatting will raise an error. Check any existing translations for reference (same filename same line number), better non-familar language.

File: locales/es/front/credits.json (hunk 1 above)
Comment (antoromani) [reply locales/es/front/credits.json]: Hello, I made the corrections.

File: locales/es/front/receipts.json (hunk 2 above)
Comment (antoromani) [reply locales/es/front/receipts.json]: I also reviewed 4 more files. Thank you for the indications

**Comment by Vovan-VE** [comment]
@antoromani , Thanks for contribution! Sorry for delay, I will check it as soon as possible.

**Comment by Vovan-VE** [comment]
@antoromani , language published. Thanks for contribution! Feel free to collect remarks, if you will find some, to open new PR later.

**Pull request merged**